doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -339,7 +339,7 @@ A more detailed "add" example using curl::
 
 Example python code to add a file. This may be run by changing these parameters in the sample code:
 
-* ``dataverse_server`` - e.g. https://dataverse.harvard.edu
+* ``dataverse_server`` - e.g. https://dataverse.example.edu
```

Comment:
Would it be too much effort to use `example.dataverse.org` as an example, and then setup a redirect for that example domain to demo.dataverse.org? @pdurbin...??